### PR TITLE
correcting translations sr_Latn

### PR DESCRIPTION
**Pull request by cvele**

### CODE REVIEW

**Comment by umpirsky** [comment]
@cvele Thanks. Rebase needed. Does it works without errors now?

**Comment by pilot** [comment]
rebase please

**Comment by cvele** [comment]
rebased

File: Resources/translations/time.nl.xliff [hunk 1]
```diff
@@ -56,4 +56,4 @@
             </trans-unit>
         </body>
     </file>
-</xliff>
+</xliff>
```
Comment (pilot) [inline Resources/translations/time.nl.xliff]: do not touch this file

File: Resources/translations/time.sr_Latn.xml [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+                <source>diff.in.month</source>
+                <target>{1}za 1 mesec|]2,5[za %count% meseca|]6,Inf[za %count% meseci</target>
+            </trans-unit>
+            <trans-unit id="13">
+                <source>diff.in.year</source>
+                <target>{1}za 1 godinu|]2,Inf[za %count% godine</target>
+            </trans-unit>
+        </body>
+    </file>
+</xliff>
```
Comment (pilot) [inline Resources/translations/time.sr_Latn.xml]: add new line pls

File: Resources/translations/time.nl.xliff (hunk 1 above)
Comment (cvele) [inline Resources/translations/time.nl.xliff]: Was just correcting the conflict that appeared during rebase, no idea why it did. But those two files are the same as you can see.

File: Resources/translations/time.sr_Latn.xml (hunk 2 above)
Comment (cvele) [inline Resources/translations/time.sr_Latn.xml]: added

**Pull request merged**